spec/built-in-modules/math.md

```diff
@@ -486,4 +486,25 @@ This function is also available as a global function named `percentage()`.
 random($limit: null)
 ```
 
-This function is also available as a global function named `random()`.
+The `random()` function can take an optional parameter `$limit` which defaults
+to `null`.
+
+* If `$limit` is `null` then return a pseudo-random floating-point number in the
+  range `[0, 1)`.
```

Comment:
A floating point number isn't a Sass number—this should be "a unitless number whose value is ..."